README.md

```diff
@@ -54,13 +54,15 @@ We provide a `Dockerfile` to build a Docker image for PyMC-Marketing so that is
 
 ## In-depth Bayesian Marketing Mix Modeling (MMM) in PyMC
 
-Leverage our Bayesian MMM API to tailor your marketing strategies effectively. Based on the research [Jin, Yuxue, et al. “Bayesian methods for media mix modeling with carryover and shape effects.” (2017)](https://research.google/pubs/pub46001/),  and integrating the expertise from core PyMC developers, our API provides:
+Leverage our Bayesian MMM API to tailor your marketing strategies effectively. Leveraging on top of the research article [Jin, Yuxue, et al. “Bayesian methods for media mix modeling with carryover and shape effects.” (2017)](https://research.google/pubs/pub46001/),  and extending it by integrating the expertise from core PyMC developers, our API provides:
 
 - **Custom Priors and Likelihoods**: Tailor your model to your specific business needs by including domain knowledge via prior distributions.
 - **Adstock Transformation**: Optimize the carry-over effects in your marketing channels.
 - **Saturation Effects**: Understand the diminishing returns in media investments.
+- **Customize adstock and saturation functions:** You can select from a variety of adstock and saturation functions. You can even implement your own custom functions.
 - **Time-varying Intercept:** Capture time-varying baseline contributions in your model (using modern and efficient Gaussian processes approximation methods).
 - **Visualization and Model Diagnostics**: Get a comprehensive view of your model's performance and insights.
+- **Choose among many inference algorithms**: We provide the option to choose between various NUTS samplers (e.g. BlackJax, NumPyro and Nutpie). See the [example notebook]([/docs/source/notebooks/mmm/mmm_example.html](https://www.pymc-marketing.io/en/stable/notebooks/general/other_nuts_samplers.html)) for more details.
```

Comment:
Can you use full link instead of relative as that will not show up in the PyPI page: https://pypi.org/project/pymc-marketing/

Related to #715

Reply:
Ok! Fixed in https://github.com/pymc-labs/pymc-marketing/pull/735/commits/9f8028c9582a34d9349cb18e931ac4145b863663 :)